dartagnan/src/main/java/com/dat3m/dartagnan/program/analysis/alias/AliasAnalysis.java

```diff
@@ -41,6 +42,31 @@ static AliasAnalysis fromConfig(Program program, Configuration config) throws In
         return a;
     }
 
+    static boolean virtualLoc(MemEvent e1, MemEvent e2) {
+        if (!(e1.getAddress() instanceof MemoryObject) || !(e2.getAddress() instanceof MemoryObject)) {
```

Comment:
Add a TODO comment saying that for the moment we do not support virtual aliasing for pointers

Reply:
Added